plugins/orchestrator-common/src/types.ts

```diff
@@ -65,3 +72,13 @@ export enum WorkflowCategory {
   ASSESSMENT = 'assessment',
   INFRASTRUCTURE = 'infrastructure',
 }
+
+export interface WorkflowOverview {
+  workflowId: string;
+  name?: string;
+  lastTriggered: string | number;
```

Comment:
it'll be easier to handle if it's optional and always string

Reply:
[d589c85](https://github.com/caponetto/backstage-plugins/pull/36/commits/d589c85772ef1c43e03325775e17606ebe419f57)